### PR TITLE
Update misc fixes and readiness

### PR DESCRIPTION
**Pull request by watchyoursixx**
- Added Readiness to 0/20/41 SV talent builds, resets rapid fire and all relevant spells with a 1s GCD. Casts after Multi GCD ends, or if no multi selected after steady's GCD ends, recasts rapid when timer ends after first use.
- Fixed stats for Heartrazor
- Fixed itemid's for Heartless and Heartrazor
- Fixed issue where debuffs set to 100% would not properly refresh/apply
- Adjusted HM to not refresh at 98% or above causing loss of stacks. Instead just delays initial cast
- Fixed phase for Sunrage Treads
- added dps estimate for Band of the Eternal Champion for gear lists
- added imp Sanctity Aura to pet DPS
- Fixed an issue with Draenei hit racial missing from player if Draenei race selected
- Added function for debugging with startStepOnly

### CODE REVIEW

**Comment by watchyoursixx** [comment]
adjusted the dps values shown for hit to behave a little differently than full hit all the time. Basically now gives value to an item with hit up to hit cap (hit cap based on surefooted and faerie fire). If an item equipped already gives hit, will subtract that value from the current total when comparing items. So if an item gives 17 hit and you're at hit cap of 95, comparing a similar item of same slot with 17 hit will also value that 17 hit (or whatever value up to the cap).

Ex. DPS value of shadowmoon drape with a given set of EP is 97.5 at current hit of 76. Similarly, the Drape of the Dark Reavers with the same EP at 76 hit is 79.22. Equipping Shadowmoon Drape puts you at 93 hit. The EP value of both cloaks stays the same (whereas how I had it on the sheet... Shadowmoon and Drape of the Dark REavers EP would change to 79.69 and 60.97 respectively)

Tl;dr is that hit up to cap should be more closely representative of the DPS gained/lost of that specific slot. An item that says it's a 9 dps upgrade should be close to a 9 dps upgrade, regardless if it has hit on it or not
